uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
+            ng-disabled="$ctrl.isMode('view') || (!$ctrl.isNewRequest() && !$ctrl.isLeaveType('sickness'))"
```

Comment:
why is manager check removed from here? and not sure why `(!$ctrl.isNewRequest()` test is added across file here?